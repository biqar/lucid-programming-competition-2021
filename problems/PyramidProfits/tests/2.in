5
9bf970 Jude 70
14ded7 Jude 90
06af9b 9bf970 80
fd3139 9bf970 30
85272e Jude 35
5
06af9b 1011400
14ded7 1950600
06af9b 26000
fd3139 1039500
Jude 323500
