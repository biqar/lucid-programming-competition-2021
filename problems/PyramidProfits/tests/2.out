06af9b 580944
14ded7 1755540
85272e 0
9bf970 654591
Jude 1141630
fd3139 218295
